src/components/Table/components/TableBody/test.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import renderer from 'react-test-renderer';
+import Table from '../../index';
+import TableBody from './index';
+import TableRow from '../TableRow';
+import Typography from '../../../Typography';
+
+
+describe('Table Body', () => {
+
+
+    it('Renders correctly', () => {
+        const content = renderer
+            .create((
+                <TableBody>
```

Comment:
Same as below - probably should just include the tableRow component and maybe some children to see that those are output correctly, but probably no other components so they don't affect these tests.

Reply:
Done.